packages/data/src/redux-store/test/index.js

```diff
@@ -233,4 +244,163 @@ describe( 'controls', () => {
 			);
 		} );
 	} );
+
+	describe( 'atomSelectors', () => {
+		const createUseSelectAtom = ( mapSelectToProps ) => {
+			return createDerivedAtom( ( { get } ) => {
+				const current = registry.__internalGetAtomResolver();
```

Comment:
Can you explain what is it doing? It might be nice to add a code comment here.

Reply:
This is basically for "registry selectors", basically I'm retrieving the `get` function passed to the parent atom.  I think I have a comment  somewhere in the registry about that.